### PR TITLE
Added an admin api endpoint

### PR DESCRIPTION
**Pull request by atrout**
Simple admin api endpoint to pass queries through from admin-web

### CODE REVIEW

File: src/apiProxy/apiConfigCreators.js [hunk 1]
```diff
@@ -71,6 +71,16 @@ export function event(params) {
 }
 
 /**
+ * admin endpoints
+ */
+export function admin(params) {
+	return {
+		endpoint,
```
Comment (mmcgahan) [inline src/apiProxy/apiConfigCreators.js]: `endpoint` is undefined here.

The current version of the platform allows you specify the `endpoint` directly in your query - no more need for changes to `apiConfigCreators`, which is now deprecated

Comment (atrout) [inline src/apiProxy/apiConfigCreators.js]: Ok, this is a little confusing. I tried to do away with the apiConfigCreator and just use the endpoint in the photosQuery, but I got an error about the type being undefined.

Comment (mmcgahan) [inline src/apiProxy/apiConfigCreators.js]: yeah, probably not using the latest meetup-web-platform - `v0.10.436`.

When you upgrade, you might get a `action$... ofType is not a function` or something like that. You'll need to `import rxjs;` at the top of your `browser-app-entry` and `server-app-entry` files to fix that, but I think the rest of the upgrade should be straightforward [more info here](https://github.com/meetup/meetup-web-platform/blob/master/CHANGELOG.md)

**Comment by atrout** [comment]
Ok, don't need this.

**Pull request closed without merging**